Transmission.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 51;
+	objectVersion = 54;
```

Comment:
This would break on older Xcode, and for Transmission 4 we need to keep compatibility. So just undo that change, thanks.